js/chatbot/src/ChatBot.svelte

```diff
@@ -82,6 +86,31 @@
 	}
 </script>
 
+{#if shareable && value !== null && value.length > 0}
+	<div class="icon-button">
+		<IconButton
+			Icon={Community}
+			label="Share"
+			on:click={async () => {
+				if (value === null) return;
+				const formatted_chat = await format_chat_for_sharing(value);
+				let title = "";
+				if (shareable === true) {
+					for (let msg of value[0]) {
+						if (msg !== null && typeof msg === "string") {
+							title = msg;
```

Comment:
For consistency with other components we should not be setting the title for the discussion post